### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
### ✅ Checklist

- [x] Github issue details are up to date for people to QA.
- [x] I have tested all my changes.

### 🕵️‍♂️ Notes for Code Reviewer

Just some cleanup of the recent code added, no major logic changes, 

Removed the empty state of the model and made it null. Its either there or not. 

### 👯‍♀️ Paired with

Solo

### CODE REVIEW

File: lib/datasource/remote/model/guardians_config_model.dart [hunk 1]
```diff
@@ -33,9 +39,7 @@ class GuardiansConfigModel {
       };
 
   void add(Account account) {
-    if (!guardians.contains(account)) {
```
Comment (gguijarro-c-chwy) [inline lib/datasource/remote/model/guardians_config_model.dart:36]: set allows only unique items.

File: lib/datasource/remote/model/guardians_config_model.dart [hunk 2]
```diff
@@ -2,22 +2,28 @@ import 'package:seeds/datasource/local/models/account.dart';
...
+
   bool get areGuardiansActive => !isEmpty;
 
   int get length => guardians.length;
+
   List<String> get guardianAddresses => guardians.map((e) => e.address).toList();
 
   GuardiansConfigModel({required this.guardians, required this.delayPeriod, required this.threshold});
 
+  factory GuardiansConfigModel.empty() {
```
Comment (gguijarro-c-chwy) [inline lib/datasource/remote/model/guardians_config_model.dart:19]: moved the static method to a factory

File: lib/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_bloc.dart [hunk 3]
```diff
@@ -59,22 +60,22 @@ class GuardiansBloc extends Bloc<GuardiansEvent, GuardiansState> {
 
   FutureOr<void> _initial(Initial event, Emitter<GuardiansState> emit) async {
     emit(state.copyWith(pageState: PageState.loading));
-    final result = await _getGuardiansDataUseCase.getGuardiansData();
+    final Result<GuardiansConfigModel> result = await _getGuardiansDataUseCase.getGuardiansData();
 
-    // TODO(n13): Should use mapper
-    final guardiansModel = (result.isValue && result.asValue!.value != GuardiansConfigModel.empty)
```
Comment (gguijarro-c-chwy) [inline lib/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_bloc.dart:65]: this is not needed, the service returns an error or a model. NO need to create another model

File: lib/datasource/remote/polkadot_api/polkadot_repository.dart [hunk 4]
```diff
@@ -301,11 +301,12 @@ class PolkadotRepository extends KeyRepository {
         res['address'] = address;
         guardiansModel = GuardiansConfigModel.fromJson(res);
       } else {
-        guardiansModel = GuardiansConfigModel.empty;
+        return Result.error(ErrorResult("Error Loading Guardians"));
       }
-      return Future.value(Result.value(guardiansModel));
```
Comment (gguijarro-c-chwy) [inline lib/datasource/remote/polkadot_api/polkadot_repository.dart:306]: future not needed

Comment (gguijarro-c-chwy) [reply lib/datasource/remote/polkadot_api/polkadot_repository.dart:306]: this returns error or model.

Comment (n13) [reply lib/datasource/remote/polkadot_api/polkadot_repository.dart:306]: In my mind there were 3 cases:
- Guardians load
- Guardians load but user hasn't set any - I am not entirely sure what that returns from the polkadot call. Need to test it. 
- Load error (of any kind)

File: lib/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_bloc.dart (hunk 3 above)
Comment (n13) [reply lib/screens/profile_screens/guardians/guardians_tabs/interactor/viewmodels/guardians_bloc.dart:65]: But... the initial state when there is no model on the server should be empty model, no? So we can start filling it up...

**Review by n13 — APPROVED** [review]

**Comment by n13** [comment]
Looks good just some comments & questions

**Pull request merged**